src/Microsoft.Health.Dicom.Core/Features/Common/DefaultExternalCredentialProvider.cs

```diff
@@ -1,14 +1,15 @@
-﻿// -------------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------------
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
 using Azure.Core;
 using Azure.Identity;
+using Microsoft.Health.Core.Features.Identity;
 
 namespace Microsoft.Health.Dicom.Core.Features.Common;
 
-internal sealed class DefaultExternalOperationCredentialProvider : IExternalOperationCredentialProvider
+internal sealed class DefaultExternalCredentialProvider : IExternalCredentialProvider
```

Comment:
Don't we have this type in core? Do we have a method to register it as a service?

Reply:
Added registration method (previously it was only a part of CMK registration)